test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -669,6 +677,29 @@ def test_successful_auth_and_capture_with_gateway_specific_recurring_stored_cred
     assert_success capture
   end
 
+  def test_successful_recurring_purchase_with_apple_pay_credentials
+    stored_credential_params = stored_credential(:initial, :recurring, :merchant)
+    assert auth = @gateway.authorize(@amount, @apple_pay_network_token, @options.merge({ stored_credential: stored_credential_params, wallet_type: :apple_pay }))
```

Comment:
`wallet_type` is not related to apple pay and would not be set if the payment method is apple pay